tsiguenz/ft_transcendence
# Front chat social

Add the chat social front and changed some issues from friends.

To test :
-  join a room, send a msg, leave the room.
- Add a friend from leaderboard, go to social > my friends > delete it. 
- Watch your connected friends. When remove a connected friend from  social > my friends, you should refresh the page to make it disappear. (need help to resolve it)
